cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionedFileSetArguments.java

```diff
@@ -130,6 +152,7 @@ public static PartitionFilter getInputPartitionFilter(Map<String, String> argume
     return builder.build();
   }
 
+  // TODO: should be private, or is it designed to be used externally?
```

Comment:
I was following hadoop style, where, for example, FileInputFormat has a method addInputPath() but also a public constant INPUT_DIR. Since this is part of API, we can hardly change it now.